projects/luajit/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Sergey Bronnikov
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM sydr/ubuntu20.04-sydr-fuzz
+
+MAINTAINER Sergey Bronnikov
+
+# Install build dependencies.
+RUN apt-get update && apt-get install -y \
+	build-essential ninja-build cmake make \
+	libreadline-dev libunwind-dev
```

Comment:
Please, remove all the dependencies except of `libunwind-dev` and `libreadline-dev`.
They are installed in the base sydr-fuzz docker container.

Reply:
@ligurio 
Base image dockerfile can be found in `oss-sydr-fuzz/docker/ubuntu20.04-sydr-fuzz/`